appinventor/docs/reference/components/sensors.html

```diff
@@ -662,43 +662,68 @@ <h2 id="LocationSensor">LocationSensor</h2>
-  <dd></dd>
+    <dd><p>Altitude of the Android device in meters, if available.</p>
+    <p>Altitude is measured from the <a target="_blank" href="https://gisgeography.com/wgs84-world-geodetic-system/">World Geodetic System 84 reference ellipsoid</a>,
+        not sea level. </p>
+    <p>Note that it is difficult for devices to accurately sense altitude. Altitude reported on a phone/tablet can
+        easily be off by 30 meters or more.</p></dd>
   <dt><code><em>AvailableProviders</em></code></dt>
-  <dd></dd>
+  <dd>List of available service providers, such as gps or network. This information is provided as a list and in text
+      form.</dd>
   <dt><code><em>CurrentAddress</em></code></dt>
-  <dd></dd>
+    <dd><p>Physical street address of the Android device from Google's map database.</p>
+    <p>The address might not always be available from the provider, and the address reported may not always be of the building where the device is located.</p>
+    <p>If Google has no address information available for a particular location, this will return No address available.</p>
+    </dd>
   <dt><code>DistanceInterval</code></dt>
-  <dd>Determines the minimum distance interval, in meters, that the sensor will try to use for sending out location updates. For example, if this is set to 5, then the sensor will fire a LocationChanged event only after 5 meters have been traversed. However, the sensor does not guarantee that an update will be received at exactly the distance interval. It may take more than 5 meters to fire an event, for instance.</dd>
+  <dd><p>Determines the minimum distance interval, in meters, that the sensor will try to use for sending out location
+      updates. For example, if this is set to 50, then the sensor will fire a LocationChanged event only after 50 meters
+      have been traversed. However, the sensor does not guarantee that an update will be received at exactly the
+      distance interval. It may take more than 5 meters to fire an event, for instance.</p>
+    <p>It is also useful to check against Accuracy when using this property. When your device is moving, the accuracy
+    of the detected location is constantly changing.</p></dd>
   <dt><code>Enabled</code></dt>
-  <dd></dd>
+  <dd>If true, the LocationSensor is reading data from the GPS. If false, it is not. This value does not affect whether
+  the device's GPS is on or off.</dd>
   <dt><code><em>HasAccuracy</em></code></dt>
-  <dd></dd>
+  <dd>If true, Android device can report its accuracy level.</dd>
   <dt><code><em>HasAltitude</em></code></dt>
-  <dd></dd>
+  <dd>If true, Android device can report its altitude.</dd>
   <dt><code><em>HasLongitudeLatitude</em></code></dt>
-  <dd></dd>
+  <dd>If true, Android device can report longitude and latitude.</dd>
   <dt><code><em>Latitude</em></code></dt>
-  <dd></dd>
+  <dd>Android device latitude. This is reported in decimal degrees to five decimal places. Latitude is a value between
```

Comment:
Minor nit, but we probably shouldn't specify "Android" device because of iOS...

Reply:
Also thinking that maybe it makes sense to do the Android removal as a second pass over all the documentation.